packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposal/clearProposal.ts

```diff
@@ -23,11 +23,11 @@ const makeContractInput = async (input: CommandInput): Promise<ContractInput> =>
 }
 
 const validateInput = (input: CommandInput): boolean => {
-  if (!input.proposalId) throw new Error('A proposal ID is required. Provide it with --id flag')
+  if (!input.proposalId) throw new Error('A Config Proposal ID is required. Provide it with --configProposal flag')
   return true
 }
 
-// yarn gauntlet ocr2:clear_proposal --network=bombay-testnet --id=7 terra14nrtuhrrhl2ldad7gln5uafgl8s2m25du98hlx
+// yarn gauntlet ocr2:clear_proposal --network=bombay-testnet --configProposal=7 terra14nrtuhrrhl2ldad7gln5uafgl8s2m25du98hlx
```

Comment:
This comment should be turned into a proper example and exposed. Please open a GH Issue for relevant situations like this if not a quick fix.